### PR TITLE
Login page

### PR DESCRIPTION
**Pull request by ishapatell1**
Screenshot (97)

### CODE REVIEW

**Comment by aqsaaqeel** [comment]
isha image
The avatar is not visible, otherwise it looks clean and nice

**Review by aqsaaqeel — APPROVED** [review]

File: Login/login.css [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+}
+.signup-page-content{
+position: absolute;
+  top: 14%;
+  left: 50%;
+  transform: translateX(-50%);
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  border: 0.1rem solid #cfcfcf;
```
Comment (way2nizam) [inline Login/login.css:16]: please use variable for color

**Review by way2nizam — COMMENTED** [review]
overall is good. 
one suggestion use variables for color.

File: Login/login.css (hunk 1 above)
Comment (ishapatell1) [reply Login/login.css:16]: Sure I will make improvize this code

**Pull request merged**